README.md

```diff
@@ -3,9 +3,7 @@
 
 <img src="https://travis-ci.org/mozilla/TTS.svg?branch=dev"/>
 
-This project is a part of [Mozilla Common Voice](https://voice.mozilla.org/en). TTS aims a deep learning based Text2Speech engine, low in cost and high in quality. To begin with, you can hear a sample generated voice from [here](https://soundcloud.com/user-565970875/commonvoice-loc-sens-attn).
-
-TTS includes two different model implementations which are based on [Tacotron](https://arxiv.org/abs/1703.10135) and [Tacotron2](https://arxiv.org/abs/1712.05884). Tacotron is smaller, efficient and easier to train but Tacotron2 provides better results, especially when it is combined with a Neural vocoder. Therefore, choose depending on your project requirements.
+This project is a part of [Mozilla Common Voice](https://voice.mozilla.org/en). TTS aims a deep learning based Text2Speech engine, low in cost and high in quality. To begin with, you can hear a sample synthesized voice from [here](https://soundcloud.com/user-565970875/commonvoice-loc-sens-attn).
```

Comment:
Maybe its time to replace this with https://github.com/mozilla/TTS/blob/0176cd6b2c673ee5a9b2d7e3226dcf1cd37a82ee/tests/data/ljspeech/wavs/LJ001-0001.wav?raw=true 
since it sound so much better.

Reply:
Good point. I'll update it with the new version.